### ISSUE TITLE
Magazine Weight Bug

### ISSUE THREAD

**Issue by RyanMcc101** [acemod/ACE3#4745]
**Arma 3 Version:** `1.66` (stable)
**CBA Version:** `3.1.2.161105` (stable)
**ACE3 Version:** `3.8.2` (stable)

**Mods:**
- `@CBA_A3`
- `@ace`

**Description:**
- When a magazine is loaded in your rifle, the weight of it is doubled. 
Link below describes problem, but as an example.
If you have 9 magazines in your vest and you weigh 17.6Kg, when you load your rifle, your total weight shouldn't change.  However currently if you load your rifle, your weight increases to 18.05Kg.
[Screenshots of issues](http://imgur.com/a/kTP5F)

**Steps to reproduce:**
- Open editor
- Place rifleman
- Preview in single player.
- Move ammunition between inventory, rifle and ground to see weight changes. 

**Where did the issue occur?**
- Single Player. Multi player not tested. 

**Placed Modules:**
- N/A

**Comment by commy2** [comment]
Should be tested in vanilla using the `load` command. We don't do the weight simulation, we just show a number representing it.

**Comment by PabstMirror** [comment]
Tested in vannila with `loadAbs player` on a blu heavy gunner:
```
1 in vest, 1 in gun - 724
2 in vest, 0 in gun - 664
1 in vest, 0 in gun - 604
0 in vest, 1 in gun - 664
0 in vest, 0 in gun - 544
```
So. in gun = 120;  in vest = 60; ????

**Comment by commy2** [comment]
Does the same apply to rocket launchers? It would actually make a difference there.

**Comment by commy2** [comment]
It actually is. That's broken:
```sqf
// getUnitLoadout player
// loadAbs player

[[],["launch_NLAW_F","","","",[],[],""],[],[],[],["B_AssaultPack_rgr_LAT",[]],"","",[],["","","","","",""]]
120

[[],["launch_NLAW_F","","","",[],[],""],[],[],[],["B_AssaultPack_rgr_LAT",[["NLAW_F",1,1]]],"","",[],["","","","","",""]]
200

[[],["launch_NLAW_F","","","",["NLAW_F",1],[],""],[],[],[],["B_AssaultPack_rgr_LAT",[]],"","",[],["","","","","",""]]
280
```

**Comment by commy2** [comment]
ref: https://forums.bistudio.com/topic/151099-scripting-discussion-dev-branch/?p=3126612

**Comment by PabstMirror** [comment]
Will be fixed in arma 1.68
